app/views/layouts/application.html.erb

```diff
@@ -1,36 +1,38 @@
 <!DOCTYPE html>
 <html>
-  <head>
```

Comment:
the new indentations is not correct. It was fine in earlier version.

Reply:
Indentation has been fixed.